docs/how-to-guides/faq.md

```diff
@@ -23,6 +23,25 @@ which may introduce breaking changes to custom tool code.
 
 Reach [OpenAI migration guide](https://github.com/openai/openai-python/discussions/742) for more details.
 
+### Promptflow 1.8.0 upgrade guide
+If you are upgrading promptflow from an existing version to promptflow 1.8.0, please run:
```

Comment:
Before upgrading to promptflow version 1.8.0 or later, it's important to first uninstall any existing installations of promptflow and its sub-packages. This ensures a clean installation of the new version without any conflicts. For detailed instructions on how to uninstall promptflow, please refer to the [FAQ](scripts/installer/windows/install_from_msi.md#uninstall) of our documentation.